core/templates/dev/head/pages/exploration_editor/RouterService.js

```diff
@@ -211,7 +221,11 @@ oppia.factory('RouterService', [
       navigateToFeedbackTab: function() {
         _savePendingChanges();
         $location.path('/feedback');
-      }
+      },
+      navigateToIssuesTab: function() {
+        _savePendingChanges();
+        $location.path('/issues');
```

Comment:
Could you extract the ``/issues`` into its own variable? It seems to be referenced several times in this file, and it seems worthwhile to de-duplicate. Ditto for the other variables only if it isn't too much of an effort.